configs/foreman/nightly.yaml

```diff
@@ -98,3 +98,271 @@
+        - redhat-rpm-config
+        - rpm-build
+        - sed
+        - shadow-utils
+        - tar
+        - unzip
+        - util-linux
+        - which
+        - xz
+      srpm-build:
+        - bash
+        - redhat-release-server
+        - redhat-rpm-config
+        - rhpkg-simple
+        - rpm-build
+        - shadow-utils
+  - name: foreman-client-nightly-rhel6
+    based_off: null
+    helper_tags:
+      foreman-client-nightly-rhel6-override: null
+      foreman-client-nightly-rhel6: null
+    build_target: foreman-client-nightly-rhel6-build
+    arches:
+      - x86_64
+    inherits:
+      foreman-client-nightly-rhel6-build:
+        foreman-client-nightly-rhel6-override: 0
+      foreman-client-nightly-rhel6-override:
+        foreman-client-nightly-rhel6: 0
+      foreman-client-nightly-rhel6: {}
+    external_repos:
+      - rhel-6.6-server
+      - rhel-6.6-server-optional
+      - epel-6
+    build_groups:
+      build:
+        - bash
+        - bzip2
+        - coreutils
+        - cpio
+        - diffutils
+        - findutils
+        - gawk
+        - gcc
+        - gcc-c++
+        - grep
+        - gzip
+        - info
+        - make
+        - patch
+        - redhat-release-server
+        - redhat-rpm-config
+        - rpm-build
+        - sed
+        - shadow-utils
+        - tar
+        - unzip
+        - util-linux
+        - which
+        - xz
+      srpm-build:
+        - bash
+        - redhat-release-server
+        - redhat-rpm-config
+        - rhpkg-simple
+        - rpm-build
+        - shadow-utils
+  - name: foreman-client-nightly-rhel7
+    based_off: null
+    helper_tags:
+      foreman-client-nightly-rhel7-override: null
+      foreman-client-nightly-rhel7: null
+    build_target: foreman-client-nightly-rhel7-build
+    arches:
+      - x86_64
+    inherits:
+      foreman-client-nightly-rhel7-build:
+        foreman-client-nightly-rhel7-override: 0
+      foreman-client-nightly-rhel7-override:
+        foreman-client-nightly-rhel7: 0
+      foreman-client-nightly-rhel7: {}
+    external_repos:
+      - rhel-7.0-server
+      - rhel-7.0-server-optional
+      - epel-7
+      - centos-7-server
+    build_groups:
+      build:
+        - bash
+        - bzip2
+        - coreutils
+        - cpio
+        - diffutils
+        - findutils
+        - gawk
+        - gcc
+        - gcc-c++
+        - grep
+        - gzip
+        - info
+        - make
+        - patch
+        - redhat-release-server
+        - redhat-rpm-config
+        - rpm-build
+        - sed
+        - shadow-utils
+        - tar
+        - unzip
+        - util-linux
+        - which
+        - xz
+      srpm-build:
+        - bash
+        - redhat-release-server
+        - redhat-rpm-config
+        - rhpkg-simple
+        - rpm-build
+        - shadow-utils
+  - name: foreman-client-nightly-fedora27
+    based_off: null
+    helper_tags:
+      foreman-client-nightly-client-fedora27: null
```

Comment:
Did you also want to include `foreman-client-nightly-fedora27` here?

Reply:
I am guessing you meant did I want an override tag here. That's a fair question. Will add.